doc/cephfs/nfs.rst

```diff
@@ -79,3 +79,234 @@ Current limitations
 
 - Per running ganesha daemon, FSAL_CEPH can only export one Ceph file system
   although multiple directories in a Ceph file system may be exported.
+
+Exporting over NFS clusters deployed using rook
+===============================================
+
```

Comment:
As a broad point, I think it might be better to reorganize this information. It seems like we ought to have a series of docs on deploying ceph using rook, and then just add a "chapter" on adding NFS gateways.